website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,125 @@ title: Setup Secrets Operator
 hide_title: true
 ---
 
+import TierLabel from "../_components/TierLabel";
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
 # Setup Secrets Operator
 
-// Explain that WGE utilize existing operators to manage secrets across fleet of clusters, and the current supported operators are ESO and SOPS.
+Platform operators face a lot of challenges in bootstrapping and managing secrets in Kubernetes across multiple clusters. 
+Specially when storing Kubernetes Secrets in `Git` safely and securely.
+To achive this, WGE has two approches to work with kubernetes secrets.
+
+1- [External Secrets Operator](https://external-secrets.io/v0.8.1/)
+
+2- [Mozilla Sops](https://fluxcd.io/flux/guides/mozilla-sops/)
 
 ## Set up ESO
-// Explain how to use the ESO profile to set up ESO on management and leaf clusters.
+
+WGE installs ESO as a [profile](https://github.com/weaveworks/profiles-catalog/tree/main/charts/external-secrets)
+
+The profile consists of 2 components
+
+1- [The HelmChart for External Secrets Operator](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/Chart.yaml)  
+- This will install the operator helm chart and the operator CRDs. more info [here](https://external-secrets.io/v0.8.1/examples/gitops-using-fluxcd/)
+
+2- [Kustomization Reference to secret stores CRs](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/templates/secret-stores-kustomization.yaml)
+
+- This kustomization refers to the git repository with the secrets and can have the values from [values file](https://github.com/weaveworks/profiles-catalog/blob/main/charts/external-secrets/values.yaml) according to your secrets git repository
+
+### Installation Steps
+
+- [Bootstrap the required secrets](/secrets/bootstraping-secrets.mdx) for the Profile Helm Repository and the Secrets Repository
+- Install the [Profile Helm repository](https://docs.gitops.weave.works/docs/gitops-templates/cli/#profiles)
+- To install the operator on Management Cluster
+    - Go to `Applications` from the side bar and `Add new application`
+    - Select `External Secrets` and input your `namespace`
+        ![ESO Profile](./imgs/eso1.jpg)
+    - Then Adjust the `values` according to your secrets git repository
+        ![ESO Values](./imgs/eso2.jpg)
+    - After that Authenticate with Github, Create and merge the pull request
+        ![ESO PR](./imgs/eso3.jpg)
+    - Once Flux reconcile the git source this should take few minutes, you will find the operator installed and the secrets lives in the cluster
+- To bootstrap the operator on a leaf Cluster
+    - Go to `Clusters` from the side bar and `Create A Cluster`
+    - Select your template (must have the enable profiles annotation)
+        ![ESO Cluster](./imgs/eso5.jpg)
+    - Select `External Secrets` from and input your `namespace`
+        ![ESO Profile](./imgs/eso4.jpg)
```

Comment:
I dont feel that we need to repeat  the same steps again especially with all the screenshots and everything, after the selection point you can just say something like `configure your values and create the PR as mentioned in the previous section` and remove the last three steps.